engine/access/rpc/backend/backend_transactions.go

```diff
@@ -657,10 +658,7 @@ func (b *backendTransactions) getTransactionResultFromExecutionNode(
 
 	resp, err := b.getTransactionResultFromAnyExeNode(ctx, execNodes, req)
 	if err != nil {
-		if status.Code(err) == codes.NotFound {
```

Comment:
previously, we were failing early if the TX wasn't found on any EN. This made sense when we only queried the ENs that had reported an `ExecutionResult` for the block. Now that we query any EN, this should be relaxed.